extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -193,6 +193,7 @@ function ($, w, mw, loader, nirvana, tracker, labeling) {
 	function updateEntryPointLabel(templateType) {
 		$typeLabel
 			.data('type', mw.html.escape(templateType))
+			.children().first()
```

Comment:
I think it's better to add class to first element and use `find('.className')` function here.